### PR TITLE
Added Indonesian translation

### PR DESCRIPTION
**Pull request by IrennaLumbuun**
feature: Added Indonesian transalation

### Technical

I commented out strings that haven't been translated yet. Will continue to add more transalations

### Stakeholders
@cdrini

### CODE REVIEW

**Comment by scottbarnes** [comment]
I am trying to move the two Indonesian translation PRs forward. I added @cunninghunter's changes from #8157, ran `/scripts/i18n-messages update id`, fixed some typos, and ran the file through `poedit`.

File: openlibrary/i18n/id/messages.po [hunk 1]
```diff
@@ -0,0 +1,5477 @@
...
+msgstr "Maaf. Ada masalah dengan halaman yang sedang Anda lihat."
+
+#: internalerror.html:15
+#, python-format
+msgid ""
+"We've noted the error %s and will look into it as soon as possible. Head for <a "
+"href=\"/\">home</a>?"
+msgstr ""
+"Kami telah menemukan error %s dan akan memeriksanya segera. Kembali ke <a href=\"/"
+"\">home</a>?"
```
Comment (tfmorris) [inline openlibrary/i18n/id/messages.po:242]: Is the English word "home" used for web homepages in Indonesian, or is this a missed translation?

Comment (scottbarnes) [reply openlibrary/i18n/id/messages.po:242]: @IrennaLumbuun, are you able to comment on this? I'd dearly like to get this PR into a place where we can merge it, even if it is incomplete and not ready to enable on the main site, if only so it's easier to review when changes are added in the future.

Comment (IrennaLumbuun) [reply openlibrary/i18n/id/messages.po:242]: Looks like I missed this when making the change. Let me get back on it in a couple days! Thx for following up :)

Comment (tfmorris) [reply openlibrary/i18n/id/messages.po]: Thanks for the quick response!

Comment (IrennaLumbuun) [reply openlibrary/i18n/id/messages.po]: No worries! I also marked this PR as ready so it can be merged

**Review by cdrini — APPROVED** [review]
Lgtm! Nice work folks, thank you for following up @scottbarnes and pushing to completion! 🥳 

Note these will be available to Indonesian patrons by default next time we do a deploy.

**Pull request merged**